### PR TITLE
Ben's branch

### PR DESCRIPTION
**Pull request by benchanjamin**

### CODE REVIEW

File: app/templates/activity_modal.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                        <span aria-hidden="true">&times;</span>
+                    </button>
+                </h5>
+            </div>
+            <div class="modal-body">
+                <table>
+                    <tr style="border-bottom: 1px solid #000;border-color: white">
+                        <td>
+                            <span style="display: table;margin: 0 auto;">
+                            Number of Free Food Events All-Time
```
Comment (nicholaspad) [inline app/templates/activity_modal.html:28]: just wondering, do you plan to initialize counters such as this to 0? or have you been tracking these data all along?

**Review by nicholaspad — APPROVED** [review]
nice changes!

File: app/templates/activity_modal.html (hunk 1 above)
Comment (benchanjamin) [reply app/templates/activity_modal.html:28]: There haven't been any free food events posted by others since launch, so it's still zero!

**Comment by benchanjamin** [comment]
A staging app setup would be very helpful!

**Comment by nicholaspad** [comment]
also, you might want to take a look at the logs for the new staging app - `heroku logs --tail -a food4uprinceton-stage`. there seems to be an error related to sqlalchemy.

**Comment by daphne-barretto** [comment]
> @daphne-barretto in general, we should avoid deploying to prod from a non-merged branch. please let us know if you'd like to get a staging app setup, which can be pushed to whenever
@nicholaspad I have not tried to actively deploy anything recently, and definitely not any of the times I see in this branch. I am unsure why it says I have. I have searched my GitHub and Heroku accounts to see if there's something automatically set up, but I haven't found anything yet.

**Comment by nicholaspad** [comment]
@daphne-barretto hmm, that's weird haha. hopefully they fix this issue! thanks for letting me know.

**Comment by benchanjamin** [comment]
@nicholaspad Thank you for your help! I tried running the command `heroku logs --tail -a food4uprinceton-stage` in my terminal but in return I got`›   Error: You do not have access to the app food4uprinceton-stage.
  ›
  ›   Error ID: forbidden`. Perhaps the permissions need to be changed?

**Comment by nicholaspad** [comment]
Sorry about that, should be fixed now. I added the below emails as collaborators to the staging app:
image

**Pull request merged**